src/main/java/com/toofifty/easyblastfurnace/EasyBlastFurnaceConfig.java

```diff
@@ -10,336 +10,357 @@
+	)
+	String guidanceOverlays = "guidanceOverlays";
+	@ConfigItem(
+		position = 0,
+		keyName = "showStepOverlay",
+		name = "Show step overlay",
+		description = "Show an instructional overlay of the next step",
+		section = guidanceOverlays
+	)
+	default boolean showStepOverlay()
+	{
+		return true;
+	}
+
+
+	@ConfigItem(
+		position = 1,
+		keyName = "itemOverlayMode",
+		name = "Item overlay mode",
+		description = "Select how highlighted items appear",
+		section = guidanceOverlays
+	)
+	default ItemOverlaySetting itemOverlayMode()
+	{
+		return ItemOverlaySetting.BOX;
+	}
+
+	@ConfigItem(
+		position = 2,
+		keyName = "itemOverlayTextMode",
+		name = "Item overlay tooltip",
+		description = "Select where to display tooltip text for items",
+		section = guidanceOverlays
+	)
+	default HighlightOverlayTextSetting itemOverlayTextMode()
+	{
+		return HighlightOverlayTextSetting.BELOW;
+	}
+
+	@ConfigItem(
+		position = 3,
+		keyName = "itemOverlayColor",
+		name = "Item overlay color",
+		description = "Change the color of the item overlay",
+		section = guidanceOverlays
+	)
+	default Color itemOverlayColor()
+	{
+		return Color.CYAN;
+	}
+
+	@ConfigItem(
+		position = 4,
+		keyName = "showObjectOverlays",
+		name = "Show object overlays",
+		description = "Enables clickbox overlays for the next object to click",
+		section = guidanceOverlays
+	)
+	default boolean showObjectOverlays()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		position = 5,
+		keyName = "useDepositInventory",
+		name = "Use deposit inventory button",
+		description = "When enabled, the deposit inventory button will be highlighted instead of individual items in your inventory during bank deposits.",
+		section = guidanceOverlays
+	)
+	default boolean useDepositInventory()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		position = 6,
+		keyName = "objectOverlayTextMode",
+		name = "Object overlay tooltip",
+		description = "Select where to display tooltip text for game objects",
+		section = guidanceOverlays
+	)
+	default HighlightOverlayTextSetting objectOverlayTextMode()
+	{
+		return HighlightOverlayTextSetting.ABOVE;
+	}
+
+	@ConfigItem(
+		position = 7,
+		keyName = "objectOverlayColor",
+		name = "Object overlay color",
+		description = "Change the color of the object overlay",
+		section = guidanceOverlays
+	)
+	default Color objectOverlayColor()
+	{
+		return Color.CYAN;
+	}
+
+	@ConfigItem(
+		position = 8,
+		keyName = "leaveBarInDispenser",
+		name = "Toggle tick perfect method",
+		description = "Continues to the next step when there is 1 bar in the dispenser rather than 0",
+		section = guidanceOverlays
+	)
+	default boolean tickPerfectMethod() { return false; }
+
+	@ConfigItem(
+		position = 9,
+		keyName = "barsInDispenser",
+		name = "Configure bar quantity in dispenser",
+		description = "Number of bars in the dispenser to ignore to continue to next step for tick perfect method",
+		section = guidanceOverlays
+	)
+	default int barsInDispenser() { return 27; }
```

Comment:
can you please rename this variable to totalBarsToKeepInDispenser to help with readability.